tests/conformance/prerun.sh

```diff
@@ -7,15 +7,23 @@
 set -e
 
 FRAMEWORK_VERSION=$1
-if [ -z "${FRAMEWORK_VERSION}" ]
-    then
-        echo "Functions Framework version required as first parameter"
-        exit 1
+if [ -z "${FRAMEWORK_VERSION}" ]; then
+    echo "Functions Framework version required as first parameter"
+    exit 1
+fi
+
+PYTHON_VERSION=$2
+if [ -z "${PYTHON_VERSION}" ]; then
+    echo "Python version required as second parameter"
+    exit 1
 fi
 
 SCRIPT_DIR=$(realpath $(dirname $0))
 
 cd $SCRIPT_DIR
 
-echo "git+https://github.com/GoogleCloudPlatform/functions-framework-python@$FRAMEWORK_VERSION#egg=functions-framework" > requirements.txt
-cat requirements.txt
\ No newline at end of file
+echo "git+https://github.com/GoogleCloudPlatform/functions-framework-python@$FRAMEWORK_VERSION#egg=functions-framework" >requirements.txt
+cat requirements.txt
+
+echo $PYTHON_VERSION >.python-version
```

Comment:
Is this required for the test to use the right python version? Was the `builder-runtime` env not being used for this purpose?

Reply:
For some reason it was not picking up the correct version. See https://github.com/GoogleCloudPlatform/functions-framework-python/actions/runs/5073551086/jobs/9112706450#step:6:493 for example of the builder getting and using python v 3.11 when the action was asking for python37 . See https://github.com/GoogleCloudPlatform/functions-framework-python/actions/runs/5480718756/jobs/9984237010?pr=255#step:6:287 on example of how after adding file it is using the right version. 

Not sure if we have been always testing with latest python version or if this bug appeared later on.

Reply:
I was able to reproduce by just running builder in local function

Steps to repro:

1. `mkdir ~/my-repro cd ~/my-repro`
2.  main.py 
```
import functions_framework

@functions_framework.http
def hello(request):
    return "Hello world!"
```